akka-remote/src/main/mima-filters/2.6.1.backwards.excludes

```diff
@@ -0,0 +1,10 @@
+#28281 Remove usage of ChannelLocalActor
```

Comment:
(fyi we've been moving to making `2.6.1.backwards.excludes` a folder rather than a file to avoid some PR conflicts)

Reply:
Okay，I will update then